lib/src/signed_video_h26x_auth.c

```diff
@@ -706,8 +706,9 @@ prepare_for_validation(signed_video_t *self)
       // TODO: Is it possible to avoid a memcpy by using a pointer strategy?
       memcpy(signature_info->hash, self->gop_info->gop_hash, HASH_DIGEST_SIZE);
     }
-    SVI_THROW_IF_WITH_MSG(gop_state->signing_present && !gop_state->has_public_key, SVI_UNKNOWN,
-        "No public key found");
+
+    SVI_THROW_IF_WITH_MSG(
+        gop_state->signing_present && !self->has_public_key, SVI_UNKNOWN, "No public key found");
```

Comment:
"No public key present" since we do not search for it here. That's done elsewhere.